src/Adapter/Order/QueryHandler/GetOrderForViewingHandler.php

```diff
@@ -235,14 +235,14 @@ public function handle(GetOrderForViewing $query): OrderForViewing
     private function getOrderCustomer(Order $order, OrderInvoiceAddressForViewing $invoiceAddress): ?OrderCustomerForViewing
     {
         $currency = new Currency($order->id_currency);
-        $customer = new Customer($order->id_customer);
         $genderName = '';
         $totalSpentSinceRegistration = null;
 
         if (!Validate::isLoadedObject($customer)) {
-            $this->buildFakeCustomerObject($order, $invoiceAddress, $customer);
+            $customer = $this->buildFakeCustomerObject($order, $invoiceAddress, $customer);
```

Comment:
```suggestion
            $customer = $this->buildFakeCustomerObject($order, $invoiceAddress);
```

Reply:
I don't know what you're talking about :roll_eyes: